### ISSUE TITLE
Entire modpack broke for no reason

### ISSUE THREAD

**Issue by Fishyboye** [DarkPacks/SkyFactory-4#2172]
## Bug Description

I loaded the modpack through forge after getting the hopper bonsai and everything was broken. World presets werent there anymore, none of the advancements were there and the prestige tree was reset.

### Did this behavior use to work in the previous version?

Only used one version and didnt notice any updates

## Steps to Reproduce (for bugs)

1. Dont know what happened
2.
3.

## Logs

* Client/Server Log:
* Crash Log:

## World Information
* World Preset:
* Prestige Enabled:
* Modpack Version world created in:
* Additional Content Installed:

## Client Information
* Modpack Version:
* Java Version:
* Launcher Used:
* Memory Allocated:
* Server/LAN/Single Player:
* Optifine Installed:
* Shaders Enabled:

**Comment by ChaosAgent202** [comment]
The same exact thing happened here, and it stayed happening after I uninstalled and reinstalled it.

**Comment by SlumpyPanda** [comment]
I am also having issues loading the modpack after i uninstaled and reinstaled it

**Comment by whemmingsson** [comment]
Very interesting, this happened to me as well, so came here to look for guidance. I can provide some more details:

Yes, the world presents were all changed:
![New_world](https://user-images.githubusercontent.com/37114315/200398578-5c73efbc-88c5-4ba8-bf87-8ad54c9192c2.PNG)

The Advancements book is reset to something that looks like Stoneblock (maybe?) 
![Advancements](https://user-images.githubusercontent.com/37114315/200398604-4f304dee-bdda-42b9-afd8-9f4fce508b14.PNG)

I got a Digital Guide (for the simple generators) when respawning in the world. 
![simple_gen_guide](https://user-images.githubusercontent.com/37114315/200398662-dd1add52-0c7e-487b-829e-995ee251c9ee.PNG)

The font and/or colors seems off when loading the world. 

Got the following Forge errors when loading the world:
![missing_blocks_items](https://user-images.githubusercontent.com/37114315/200398692-a14e1596-595d-433a-8d66-15cee8e480cc.PNG)

And the following errors in-game:
![HorsePower_config_errors](https://user-images.githubusercontent.com/37114315/200398712-84b9a34c-475a-4593-ba33-836b0b13ce7d.PNG)

Also got this interesting thingy when loading the world:
![MPUtils](https://user-images.githubusercontent.com/37114315/200398792-4a8dc868-4b2f-4242-9eb5-634723f40b8c.PNG)

**Comment by ChaosAgent202** [comment]
Yeah, exactly!

**Comment by LenderSchlender** [comment]
I'm also facing a similar issue.

**Comment by YogurtInc** [comment]
Yeah, same happened to me. I played it 4 hours ago and it was fine.

**Comment by YogurtInc** [comment]
It looks like CurseForge problem.

**Comment by whemmingsson** [comment]
Yes, the issue is beeing talked about over at Darkosto's twitch.

**Comment by LenderSchlender** [comment]
It seems to work for me again, looks like they fixed it!

**Comment by mmpadi3d** [comment]
Still no workie for me.

**Comment by ChaosAgent202** [comment]
> It seems to work for me again, looks like they fixed it!

What did you do? Its still off for me

**Comment by eyelesa** [comment]
Still broken for me as of 5pm est. Picked up SkyFactory again for the first time in years and it broke a day and a half in, worst timing ever lol. Is there any indication that its being worked on?

**Comment by LenderSchlender** [comment]
> > It seems to work for me again, looks like they fixed it!
> 
> What did you do? Its still off for me

I used the global repair option that appears when you click on the three dots at the top right, first it said it couldn't do it because minecraft was running, but after a restart it worked!

**Comment by woDeee** [comment]
> > > It seems to work for me again, looks like they fixed it!
> > 
> > 
> > What did you do? Its still off for me
> 
> I used the global repair option that appears when you click on the three dots at the top right, first it said it couldn't do it because minecraft was running, but after a restart it worked!

Ok worked for me too, but if i close minecraft and close CurseForge and restart both, they updating minecraft again and its broken again. I need to repair it every time, i start SkyFactory. There is something broken !

**Comment by GrumpyUrsus** [comment]
basically it's anything to do with the mod that works with the creation of blocks using resin and acorns from the trees near the beginning of the game. so no cobblestone no dirt no clay etc etc etc. at least with me anyways.

**Comment by potato1234x** [comment]
same here, all modpack exclusive content is gone

**Comment by Fishyboye** [comment]
Just found out its a curseforge issue. All modpacks from there are broken. Looked at the curseforge reddit and found out its due to an update to the minecraft launcher. Seems we just gotta wait until they fix this.
